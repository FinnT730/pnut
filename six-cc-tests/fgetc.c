int main() {
  int f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
//  putchar(f+48);
  while ((c = fgetc(f)) != -1) {
    putchar(c);
  }
  fclose(f);
}
