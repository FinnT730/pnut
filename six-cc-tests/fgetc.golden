int main() {
  int_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while (c = fgetc(f)) {
    putchar(c);
  }
}
